src/components/TokenBalance.tsx

```diff
@@ -87,20 +92,22 @@ function TokenBalance({
         <Image source={{ uri: tokensWithUsdValue[0].imageUrl }} style={styles.tokenImg} />
         <View style={styles.column}>
           <Text style={style} testID={'TotalTokenBalance'}>
-            {localCurrencySymbol}
-            {totalTokenBalanceLocal?.toFormat(2) ?? '-'}
-          </Text>
-          <Text style={styles.tokenBalance}>
-            {formatValueToDisplay(tokenBalance)} {tokensWithUsdValue[0].symbol}
+            {!hideBalance && localCurrencySymbol}
+            {hideBalance ? 'XX.XX' : totalTokenBalanceLocal?.toFormat(2) ?? '-'}
```

Comment:
This should be dependent on the decimal separator similar to the balance. If the decimal seperator is a `,` it should be `XX,XX` instead.

https://github.com/valora-inc/wallet/assets/26950305/52f9432b-9009-460a-a644-79995769d5c5